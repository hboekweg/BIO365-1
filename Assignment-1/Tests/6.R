source("6.R")
