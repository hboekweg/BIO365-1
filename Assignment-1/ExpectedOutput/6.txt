[1] 15.5
[1] 4
[1] 3
[1] 4.5
[1] 3.162276
[1] 2.772589
[1] 4
